aminator/plugins/provisioner/apt.py

```diff
@@ -47,23 +47,15 @@ class AptProvisionerPlugin(BaseProvisionerPlugin):
     """
     _name = 'apt'
 
-    def _refresh_repo_metadata(self):
-        return self.apt_get_update()
-
     @cmdsucceeds("aminator.provisioner.apt.provision_package.count")
     @cmdfails("aminator.provisioner.apt.provision_package.error")
     @lapse("aminator.provisioner.apt.provision_package.duration")
     def _provision_package(self):
-        result = self._refresh_repo_metadata()
-        if not result.success:
-            log.critical('Repo metadata refresh failed: {0.std_err}'.format(result.result))
-            return result
         context = self._config.context
         os.environ['DEBIAN_FRONTEND'] = 'noninteractive'
-        if context.package.get('local_install', False):
-            return self.apt_get_localinstall(context.package.arg)
-        else:
-            return self.apt_get_install(context.package.arg)
+        return self.install(context.package.arg,
```

Comment:
Simplified this with a straight forward install() method and param to determine localinstall or not. Stripped out metadata refresh foo as it's now handled inside the installer(s) since localinstall will want to refresh at a different point